cypress/e2e/LanguageSelection/lang_menu.js

```diff
@@ -0,0 +1,24 @@
+/// <reference types="cypress" />
+
+const { homePage, url } = require("../../support/selectors");
+
+describe('Make sure BBC.com loads properlly', () => {
+
+    it('Test Case 32.1 Test language dropdown Portugal', () => {
+        cy.get(homePage.languageDropbox).scrollIntoView().should('exist').click({force: true});
```

Comment:
sita galima persikelti i before each hooka ir parasyti viena kart, nereiktu to paties rasyti visose testuose

Reply:
fixed